### PR TITLE
Remove v1.6 from docs

### PR DESCRIPTION
**Pull request by matthewshaver**
## What are you changing in this pull request and why?

Removes v1.6 from the dropdown.
Removes content where v1.6 was "lastversion"
Cleans up "firstversion="1.7" blocks
Removes appropriate page version blocks

## Checklist
- [ ] I have reviewed the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) so my content adheres to these guidelines.
- [ ] The topic I'm writing about is for specific dbt version(s) and I have versioned it according to the [version a whole page](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) and/or [version a block of content](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#versioning-blocks-of-content) guidelines.
- [ ] I have added checklist item(s) to this list for anything anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."
- [ ] The content in this PR requires a dbt release note, so I added one to the [release notes page](https://docs.getdbt.com/docs/dbt-versions/dbt-cloud-release-notes).

---
🚀 Deployment available! Here are the direct links to the updated files:

- https://docs-getdbt-com-git-eol-dbt-labs.vercel.app/docs/build/environment-variables
- https://docs-getdbt-com-git-eol-dbt-labs.vercel.app/docs/build/metricflow-time-spine
- https://docs-getdbt-com-git-eol-dbt-labs.vercel.app/docs/build/packages
- https://docs-getdbt-com-git-eol-dbt-labs.vercel.app/docs/build/saved-queries
- https://docs-getdbt-com-git-eol-dbt-labs.vercel.app/docs/build/semantic-models
- https://docs-getdbt-com-git-eol-dbt-labs.vercel.app/docs/collaborate/govern/project-dependencies
- https://docs-getdbt-com-git-eol-dbt-labs.vercel.app/docs/core/connect-data-platform/about-core-connections
- https://docs-getdbt-com-git-eol-dbt-labs.vercel.app/docs/core/connect-data-platform/spark-setup
- https://docs-getdbt-com-git-eol-dbt-labs.vercel.app/reference/artifacts/run-results-json
- https://docs-getdbt-com-git-eol-dbt-labs.vercel.app/reference/commands/cmd-docs
- https://docs-getdbt-com-git-eol-dbt-labs.vercel.app/reference/commands/deps
- https://docs-getdbt-com-git-eol-dbt-labs.vercel.app/reference/commands/init
- https://docs-getdbt-com-git-eol-dbt-labs.vercel.app/reference/dbt_project.yml
- https://docs-getdbt-com-git-eol-dbt-labs.vercel.app/reference/global-configs/resource-type
- https://docs-getdbt-com-git-eol-dbt-labs.vercel.app/reference/node-selection/methods
- https://docs-getdbt-com-git-eol-dbt-labs.vercel.app/reference/resource-configs/access
- https://docs-getdbt-com-git-eol-dbt-labs.vercel.app/reference/resource-configs/bigquery-configs
- https://docs-getdbt-com-git-eol-dbt-labs.vercel.app/reference/resource-configs/contract
- https://docs-getdbt-com-git-eol-dbt-labs.vercel.app/reference/resource-configs/databricks-configs
- https://docs-getdbt-com-git-eol-dbt-labs.vercel.app/reference/resource-configs/enabled
- https://docs-getdbt-com-git-eol-dbt-labs.vercel.app/reference/resource-configs/group
- https://docs-getdbt-com-git-eol-dbt-labs.vercel.app/reference/resource-configs/meta
- https://docs-getdbt-com-git-eol-dbt-labs.vercel.app/reference/resource-configs/postgres-configs
- https://docs-getdbt-com-git-eol-dbt-labs.vercel.app/reference/resource-configs/redshift-configs
- https://docs-getdbt-com-git-eol-dbt-labs.vercel.app/reference/resource-configs/snowflake-configs
- https://docs-getdbt-com-git-eol-dbt-labs.vercel.app/reference/resource-properties/config
- https://docs-getdbt-com-git-eol-dbt-labs.vercel.app/reference/resource-properties/freshness

### CODE REVIEW

File: website/docs/docs/build/packages.md [hunk 1]
```diff
@@ -83,12 +83,8 @@ packages:
     version: [">=0.7.0", "<0.8.0"]
 ```
 
-<VersionBlock firstVersion="1.7">
-
 Beginning in v1.7, `dbt deps` "pins" each package by default. See ["Pinning packages"](#pinning-packages) for details.
```
Comment (runleonarun) [inline website/docs/docs/build/packages.md]: At what point would we get rid of "Beginning in v1.7?" Once 1.7 is no longer supported?

**Review by runleonarun — COMMENTED** [review]
I saw a few 1.6 references that might be worth removing:
* https://docs-getdbt-com-git-eol-dbt-labs.vercel.app/docs/collaborate/govern/model-versions#why-version-a-model Paragraph beginning with "dbt Core 1.6 introduced"
* https://docs-getdbt-com-git-eol-dbt-labs.vercel.app/best-practices/clone-incremental-models
* https://docs-getdbt-com-git-eol-dbt-labs.vercel.app/docs/collaborate/govern/project-dependencies#prerequisites (maybe just supported version of dbt" is good enough here)

Otherwise it looks good!

File: website/docs/docs/build/packages.md (hunk 1 above)
Comment (matthewshaver) [inline website/docs/docs/build/packages.md]:
```suggestion
`dbt deps` "pins" each package by default. See ["Pinning packages"](#pinning-packages) for details.
```

**Comment by matthewshaver** [comment]
> I saw a few 1.6 references that might be worth removing:
> 
> * https://docs-getdbt-com-git-eol-dbt-labs.vercel.app/docs/collaborate/govern/model-versions#why-version-a-model Paragraph beginning with "dbt Core 1.6 introduced"
> * https://docs-getdbt-com-git-eol-dbt-labs.vercel.app/best-practices/clone-incremental-models
> * https://docs-getdbt-com-git-eol-dbt-labs.vercel.app/docs/collaborate/govern/project-dependencies#prerequisites (maybe just supported version of dbt" is good enough here)
> 
> Otherwise it looks good!

Thanks @runleonarun Anywhere it's indicating a prerequisite, I left it in. I did update some of the language to be consistent elsewhere. 

I left it in the Model Version doc because it tells a story about when we created it and why we used the feature.

**Review by runleonarun — APPROVED** [review]
Looks good! Thanks for taking a look at those suggestions!

**Pull request merged**